index.md

```diff
@@ -32,25 +32,25 @@ The project strives to create an application interface which lowers the barrier
 to use the data for visualization and/or analysis.
 
 
-Tackling the Problem
---------------------
+Our Approach
+------------
 
 The project's core is an API which can be used to query the processed data.
 For this purpose we develop a cloud service which queries the freely
 accessible data of Sentinel-5P, aggregates it, stores it in a cache and makes
 it available.
 
 
-Target Audience
----------------
+Who is it for?
```

Comment:
I'm no native speaker but this sounds very informal to me.
I would rather use
```suggestion
For whom is this?
```

Reply:
@lkiesow IMO `Who is it for` is OK. I would reword to `Who is this (program/application/api/whatever) for`, but this change set as is seems fine to me.

Your suggestion sounds incredibly stuffy to me - I should be ringing the butler for a spot of tea should I use it :-). It's probably correct, but honestly I would have to look up the usage of whom to be sure. If that kind of formality is the goal then I would use your change, I think.

Reply:
Okay, fair enough.
So, let's use

```suggestion
Who is the API for?
```